src/components/molecules/form/field/index.js

```diff
@@ -28,6 +29,9 @@ const StyledField = styled.div`
     min-height: 44px;
     /* TO-DO: Tokenize this value as it's related to the other heights of components. */
   }
+  ${StyledSwitch} {
+    margin-top: 6px;
```

Comment:
This is a non standard value 😢

Reply:
I had to make this adjustment when the switch is in the Field (left-aligned) it needs this notch of margin.

Reply:
I think we need to review this in the top-align version. I'll add a big form in the overview to test all fields with top alignment.